### DISCUSSION TITLE
如何把walina迁移到disqusjs

### DISCUSSION THREAD

**Discussion by lin-snow** [walinejs/waline#982] [Q&A] [answered]
如何把walina迁移到disqusjs

**Comment by lizheming** [comment, marked as answer]
Waline 后台有导出功能，可以把数据导出出来。至于导入到 Disqus 目前还没有现成的处理程序，你可以自己转换处理一下。

**Reply by skybbx** [reply]
后台的导出功能用不了，我点击没反应。我之前用的是本地mysql,现在想换成其他免费的服务商，自己的服务不想续费了。可导出就一直出错

**Reply by lizheming** [reply]
@skybbx 我们单提个 issue 提供下相关信息，把接口的报错信息贴一下吧

**Reply by lin-snow** [reply]
> Waline 后台有导出功能，可以把数据导出出来。至于导入到 Disqus 目前还没有现成的处理程序，你可以自己转换处理一下。

主要是我也不会
